docs/src/examples/components/Menu/Usages/MenuExampleToolbar.shorthand.tsx

```diff
@@ -76,14 +77,30 @@ const items = [
     indicator: false,
     menu: {
       items: [
-        { key: '1', content: 'item1' },
+        {
+          key: '1',
+          content: 'item1',
+          icon: {
+            name: 'read-aloud',
+            variables: { outline: true },
+          },
+        },
```

Comment:
Can be just `outline: true` (#1002):
```suggestion
            outline: true,
```